app/celery/service_callback_tasks.py

```diff
@@ -36,16 +37,22 @@ def send_delivery_status_to_service(
         "created_at": status_update['notification_created_at'],
         "completed_at": status_update['notification_updated_at'],
         "sent_at": status_update['notification_sent_at'],
-        "notification_type": status_update['notification_type']
+        "notification_type": status_update['notification_type'],
+        "provider": status_update['provider'],
+        "status_reason": status_update['status_reason']
     }
+
+    # if the provider payload is found in the status_update object
+    if 'provider_payload' in status_update:
+        payload['provider_payload'] = status_update['provider_payload']
+
```

Comment:
are these are guaranteed that the key exists in status_update? just curious why we only check if one key exists.